pkg/dispatcher/operator/operator.go

```diff
@@ -33,18 +34,25 @@ type Operator struct {
 	calculationsController *calculations.Controller
 	podsController         *workers.Controller
 	redisURL               string
+	redisPW                string
 }
 
 // NewMainOperator return a new Operator
 func NewMainOperator(ctx context.Context, kubeclientset kubernetes.Interface, vegaclientset clientset.Interface, redisURL string) *Operator {
 	logger := logrus.New()
 	logger.Level = logrus.DebugLevel
+	redisPW, err := ioutil.ReadFile("redis.conf")
+	if err != nil { //wasnt able to find the path to the file/not sure where it is
+		fmt.Errorf("Failed to retrieve database password from a file: %s", err.Error())
```

Comment:
This is why the tests are failing here. By doing `fmt.Errorf` you are not calling or displaying anything here. However, I don't think that here is the right place to load the file. Please do this in the dispatcher's main and only pass the redisClient in the operator.